yb-voyager/cmd/exportData.go

```diff
@@ -163,7 +163,10 @@ func exportDataOffline() bool {
 	}
 
 	source.DB().ExportDataPostProcessing(exportDir, tablesProgressMetadata)
-	printExportedRowCount(datafile.OpenDescriptor(exportDir).TableRowCount)
+
+	tableRowCount := datafile.OpenDescriptor(exportDir).TableRowCount
+	printExportedRowCount(tableRowCount)
+	callhomeActualRowCounts(tableRowCount)
```

Comment:
How about making this function a part of callhome package? It will be something like:
```
callhome.TableRowCount(tableRowCount) ...
```